src/data/state_incentives.ts

```diff
@@ -126,6 +126,7 @@ export type DerivedFields = {
   agi_max_limit?: number;
   agi_min_limit?: number;
   authority: string;
+  geo_group?: string;
```

Comment:
Re: name bikeshedding, I don't love geo_group because if you don't know what it is already, you won't guess what it is from the name.

In my mind this relates to eligibility, so I like eligibility_group. Or if we want to emphasize that it is a particular subset of eligibility, maybe geo_eligibility_group.

Reply:
I'm a fan of `geo_eligibility_group`